packages/discovery-provider/plugins/notifications/src/types/dn.ts

```diff
@@ -695,6 +695,7 @@ export interface UsdcPurchaseRow {
   signature: string
   slot: number
   updated_at?: Date
+  splits: any
```

Comment:
can this be typed?

Reply:
This in theory should be autogenerated from sql-ts which seems to put `any` for JSONB types. I didn't know how to regenerate it so I just based it on `stream_conditions` etc

Reply:
oh lmao.... rip yuck ew. keep it autogenerated 🤷